docs/pages/toolchain/saya/persistent.md

```diff
@@ -31,6 +31,16 @@ katana init --id per1 \
 
 This will automatically deploy a fresh core contract on the settlement layer.
 
+If you want to use a specific facts registry contract, set it via the `settlement-facts-registry-contract` argument.
+
+```
+katana init --id per1 \
+    --settlement-chain <Sepolia|Mainnet> \
+    --settlement-account-address <ADDRESS> \
+    --settlement-account-private-key <PRIVATE_KEY>
+    --settlement-facts-registry-contract <ADDRESS>
```

Comment:
This is not something that is supported at the moment.
A custom option is present but may need more work to be functional.

You can however change your facts registry address directly on the deployed contract to use a mocked one for instance while developing. This can be directly changed calling the entrypoint `set_facts_registry` of Piltover.

Reply:
Ah @glihm I am submitting this PR specifically for https://github.com/dojoengine/dojo/pull/3158, of course only merged when it is accepted. The PR adds support for this feature.

I currently find this the only location that actually documents the `init` behaviour for katana cli, that's why I added this.

Sorry if I misunderstood the contribution flow, should I have waited the PR on the dojo side to be merged first?

Reply:
All good, thanks for clarifying it. I didn't see any description so wasn't sure about the focus.

Appreciate your comment and the proposal on Dojo. 👍

Reply:
> Sorry if I misunderstood the contribution flow, should I have waited the PR on the dojo side to be merged first?

Np np, yeah at least the link to the PR in dojo would have made the direct link. :)